docs/src/modules/components/Ad.js

```diff
@@ -107,7 +106,7 @@ class Ad extends React.Component {
         {adblock === true ? getAdblock(classes, t) : null}
         {adblock === false ? (
           <Tooltip id="ad-info" title={t('adTitle')} placement="left">
-            <span className={classes.info}>i</span>
+            <InfoIcon className={classes.info} />
```

Comment:
Favorite change that wasn't intended to be caught by the rule.

Reply:
I'm going back to the initial solution, it creates too much visual distraction:

![Capture d’écran 2019-06-30 à 13 50 21](https://user-images.githubusercontent.com/3165635/60396181-2efb2780-9b3e-11e9-92db-cf534af1bc3a.png)
![Capture d’écran 2019-06-30 à 13 51 30](https://user-images.githubusercontent.com/3165635/60396182-2efb2780-9b3e-11e9-980a-7275664c039a.png)

Reply:
make it smaller then?

Reply:
I can't follow this? Who told you to revert it?

Reply:
Do we need to import an icon? Can we make it simpler with a one char character `i`?